uint8 number
---
bool result